### PR TITLE
[docs] Add the Pull Request Template

### PR DESCRIPTION
**Pull request by AlexSkorkin**

### CODE REVIEW

**Comment by AlexSkorkin** [comment]
> We should add a link to
> https://github.com/DevExpress/testcafe/blob/cf74ca1c62f373b5166122b686eb8d21bf495ba3/CONTRIBUTING.md

There's a link at the top. Is it wrong?

image

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+_Describe how your changes address the issue or implement the desired functionality in as much detail as possible._
+
+## References
+_Provide a link to the existing issue(s), if any._
+
+## Pre-Merge TODO
+- [ ] Write tests for your proposed changes
+- [ ] Make sure that existing tests do not fail
+
+<!-- If your PR is not complete, mark it as [WIP] in its title -->
```
Comment (AndreyBelym) [inline .github/PULL_REQUEST_TEMPLATE.md]: And create your PR as a draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by AndreyBelym** [comment]
Well, actually a good idea is always opening your PR as a draft. But it's a very minor point, this template is quite good as it is.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (AlexSkorkin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Agree, added this info to the top of the template

**Pull request merged**